qp_qiime2/__init__.py

```diff
@@ -0,0 +1,33 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from qiita_client import QiitaPlugin, QiitaCommand
+
+from .qiime2 import rarefy
+
+__all__ = ['qiime2']
+
+
+# Initialize the plugin
+plugin = QiitaPlugin(
+    'qiime2', '4.2017', 'QIIME 2')
+
+# Define the rarefy command
+req_params = {'i-table': ('artifact', ['BIOM'])}
+opt_params = {
```

Comment:
I would put p-sampling-depth as a required parameter and not provide a default value - I think users should decide this value since it is really study dependent. Having a default value may mislead users.